apps/backend/src/tests/neighborhood_api.test.ts

```diff
@@ -36,135 +37,76 @@ beforeAll(async () => {
-      .send(BOBS_LOGIN_DATA);
-
-    const { token } = loginResponse.body;
-
-    // 1234 is invalid neighborhood id for BOBS_LOGIN_DATA
-    const deleteResponse = await api.delete('/api/neighborhoods/1234')
-      .set('Authorization', `Bearer ${token}`);
-
-    const currentNeighborhoods = await testHelpers.neighborhoodsInDb();
-    const numCurrentNeighborhoods = currentNeighborhoods.length;
-
-    expect(deleteResponse.status).toEqual(404);
-    expect(numCurrentNeighborhoods).toEqual(numInitialNeighborhoods);
-  });
-
-  test('User cannot delete neighborhood if user is not admin', async () => {
-    const initialNeighborhoods = await testHelpers.neighborhoodsInDb();
-    const numInitialNeighborhoods = initialNeighborhoods.length;
-
-    const loginResponse = await api
-      .post('/api/login')
-      .send(BOBS_LOGIN_DATA);
-    const { token } = loginResponse.body;
-
-    // 2 is invalid neighborhood id for BOBS_LOGIN_DATA with bob as admin
-    const deleteResponse = await api.delete('/api/neighborhoods/2').set('Authorization', `Bearer ${token}`);
-
-    const currentNeighborhoods = await testHelpers.neighborhoodsInDb();
-    const numCurrentNeighborhoods = currentNeighborhoods.length;
-
-    expect(deleteResponse.status).toEqual(403);
-    expect(numCurrentNeighborhoods).toEqual(numInitialNeighborhoods);
+  test('GET /neighborhoods returns 200 even if no neighborhoods were created', async () => {
+    await testHelpers.removeAllData();
+    const response: Response = await api.get('/api/neighborhoods');
+    expect(response.status).toEqual(200);
+    expect(response.body.length).toBe(0);
   });
+});
 
-  test('User cannot delete neighborhood if user is not logged in', async () => {
-    const initialNeighborhoods = await testHelpers.neighborhoodsInDb();
-    const numInitialNeighborhoods = initialNeighborhoods.length;
-
-    const deleteResponse = await api.delete('/api/neighborhoods/1');
-
-    const currentNeighborhoods = await testHelpers.neighborhoodsInDb();
-    const numCurrentNeighborhoods = currentNeighborhoods.length;
-
-    expect(deleteResponse.status).toEqual(401);
-    expect(numInitialNeighborhoods).toEqual(numCurrentNeighborhoods);
+describe('Tests for getting a single neighborhood: GET /neighborhoods/:id', () => {
+  beforeAll(async () => {
+    await seed();
   });
 
-  test('GET /neighborhoods/id existing id returns single neignborhood', async () => {
+  test('GET /neighborhoods/:id existing id returns single neighborhood', async () => {
     const loginResponse = await loginUser(BOBS_LOGIN_DATA);
     const { token } = loginResponse.body;
 
-    const neighborhood = await prismaClient.neighborhood.findFirst({
+    const neighborhood: NeighborhoodDetailsForMembers = await prismaClient.neighborhood.findFirst({
       where: { name: "Bob's Neighborhood" },
       include: {
         admin: true,
         users: true,
         requests: true,
       },
-    });
+    }) as NeighborhoodDetailsForMembers;
```

Comment:
I think the type assertion here is redundant. To avoid the error you're getting when removing it, you could add `const neighborhood: NeighborhoodDetailsForMembers | null`.